.env.example

```diff
@@ -1,4 +1,5 @@
 DBT_TEST_ATHENA_S3_STAGING_DIR=
+DBT_TEST_ATHENA_S3_TEMP_TABLE_DIR=
```

Comment:
I add this option to env to check this PR works in functional tests.
If I should remove this from `.env.sample` because it is an optional parameter, please feel free to tell me.